### ISSUE TITLE
Switch to https protocol by default

### ISSUE THREAD

**Issue by jayvdb** [xstatic-py/xstatic#6]
The code defaults to http protocol.  IMO the default in 2020 should be https.

**Comment by ThomasWaldmann** [comment]
I agree, but switching defaults can cause widespread issues. Did you check if that is the case here?

Also, it is just the default, users of xstatic can just tell to use https.

**Comment by ThomasWaldmann** [comment]
@deshipu @ReimarBauer any opinion about this?

**Comment by deshipu** [comment]
I don't think changing the defaults just for the sake of hype is wise in something that is supposed to be a stable building block. Yes, we could try checking all published code and see if this breaks anything, but I don't see the advantage for doing this in this case.

**Comment by ReimarBauer** [comment]
when that becomes changed I would prefer some "auto" feature, instead toggling it always.

**Comment by ThomasWaldmann** [comment]
the app using xstatic should usually know best what to choose, so i don't see how "auto" should do better?
